kilkari/src/main/java/org/motechproject/nms/kilkari/domain/Subscriber.java

```diff
@@ -16,16 +18,17 @@
     @Unique
     private String callingNumber;
 
-    //todo: should this be a Set<> instead of a List<>?
+    //making this a bi-directional relationship until MOTECH-1638 is fixed
```

Comment:
Comment with TODO so the build and IDE flag it.  Also should the loading be set to lazy incase we forget to change it back to one direction?